Miniforge3/construct.yaml

```diff
@@ -1,9 +1,9 @@
-{% set version = os.environ.get("MINIFORGE_VERSION", "24.11.3-1") %}
-{% set conda_libmamba_solver_version = "24.9.0"%}
+{% set version = os.environ.get("MINIFORGE_VERSION", "25.1.1-1") %}
```

Comment:
just noting for myself that this is in fact the correct version since 25.1.1-0 was previously released